### PR TITLE
[TWDAPI-207] Content age rating for mediacontroller playback.

### PR DESCRIPTION
**Pull request by m-michalski2**
Add documentation for WebAPI content age rating feature.

Signed-off-by: Michal Michalski <m.michalski2@partner.samsung.com>

### Change Description ###
+ Add MediaControllerContentAgeRating enum.
+ Add age rating attribute to playback info interface.
+ Add playback age rating setter to server interface.

### Bugs Fixed ###
N/A

### API Changes ###
ACR: http://suprem.sec.samsung.net/jira/browse/TWDAPI-207

### CODE REVIEW

File: docs/application/web/guides/multimedia/media-controller.md [hunk 1]
```diff
@@ -232,6 +232,28 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+
```
Comment (mijin85cho) [inline docs/application/web/guides/multimedia/media-controller.md]: Just asking your opinion.
From line 12 of this page, there is a list that showing main features of "Media Controller API".
(Setting up client and server pair, Managing request, ...etc)
Don't we need to add this "Setting content age rating" at that list, too?

Comment (m-michalski2) [reply docs/application/web/guides/multimedia/media-controller.md]: You are right, i have just added it.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 2]
```diff
@@ -29,6 +29,11 @@ The main features of the Media Controller API include:
 
   To [receive and handle incoming commands](#receive_custom_commands) in the server, use the `addCommandListener()` method.
 
+- Setting age an rating for currently played media
```
Comment (mijin85cho) [inline docs/application/web/guides/multimedia/media-controller.md]: Did you mean "Setting age rating for currently playing media"?

Also, please use unified word for "playing media" and "played media" between line 32, 34 and line 243.

File: docs/application/web/guides/multimedia/media-controller.md [hunk 3]
```diff
@@ -232,6 +237,30 @@ To manage the media controller features in your application, you must learn to s
 
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+
+<a name="#setting_age_rating"></a>
```
Comment (mijin85cho) [inline docs/application/web/guides/multimedia/media-controller.md]: Please change "\#setting_age_rating" to "setting_age_rating".

File: docs/application/web/guides/multimedia/media-controller.md [hunk 4]
```diff
@@ -232,6 +237,30 @@ To manage the media controller features in your application, you must learn to s
...
+
+Server can set age rating for current playback. Client can access this rating (read-only) and perform some actions such as displaying a warning for underage users.
+
+1. On the server side:
+
+    ```
+    server.updatePlaybackAgeRating("18");
+    ```
+
+2. On the client side:
```
Comment (rishu-samsung) [inline docs/application/web/guides/multimedia/media-controller.md]: Suggested edit: "Access `userAge` on the client side:"

File: docs/application/web/guides/multimedia/media-controller.md [hunk 5]
```diff
@@ -232,6 +237,30 @@ To manage the media controller features in your application, you must learn to s
...
       The `watcherId` variable stores the value, which can be used in the future to remove the listener from the server using the `removeCommandListener()` method.
 
+
+<a name="setting_age_rating"></a>
+
+## Setting content age rating for the currently playing media
+
+Server can set age rating for current playback. Client can access this rating (read-only) and perform some actions such as displaying a warning for underage users.
+
+1. On the server side:
```
Comment (rishu-samsung) [inline docs/application/web/guides/multimedia/media-controller.md]: Suggested edit: "Define minimum age on the server side:"

**Pull request merged**